### PR TITLE
Java techjobs mvc readme

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+`TestTaskThree` verifies work done to complete the `SearchController`.  This includes verifying how the `displaySearchResults` method was created and making sure that the moths calls both `findAll` and `findByColumnAndValue` methods.  This test does not check behavior.
+
+`TestTaskFour` verifies the behaviors required for `displaySearchResults` to pass data to a view.  
+
+`ApplicationTest` verifies that the application is able to run.
+ 
+### Appearance and Code Check:
+ 
+[Working Demo App](https://lc-java-3-demo.herokuapp.com/)
```
Comment (codinglikeagirl42) [inline README.md]: Update link -> still pointing to heroku

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ 
+1. Start the application.
+1. Test the search functionality of the project:
+
+    a. Initiate a search by location, using the search term "new york". Only 1 result should be returned on the page. The result may vary in appearance, but make sure it contains the job data organized similar to this image:
+
+    ![Search result](searchByLocation.png "Search Result Sample")
```
Comment (codinglikeagirl42) [inline README.md:35]: Image file seems to be missing.

**Review by codinglikeagirl42 — COMMENTED** [review]
This was a brief review of the readme with anticipation of changes in Q1. At which time, the readme will need to be resubmitted for a formal QA review.

**Review by codinglikeagirl42 — DISMISSED** [review]

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
This was a brief review of the readme with anticipation of changes in Q1. At which time, the readme will need to be resubmitted for a formal QA review.

File: README.md (hunk 1 above)
Comment (codinglikeagirl42) [reply README.md]: Please add verbiage about why students may run into issues with the app.

Comment (gildedgardenia) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (gildedgardenia) [reply README.md:35]: Fixed!

**Review by codinglikeagirl42 — APPROVED** [review]

**Pull request merged**